features/other_b/skin.feature

```diff
@@ -67,6 +73,33 @@ Feature: creating and editing skins
   When I am on skin's new page
     Then I should see "Sorry, you don't have permission"
 
+  Scenario: An admin can reject a skin
+  Given the unapproved public skin "public skin"
+    And I am logged in as an admin
+  When I go to admin's skins page
+    And I check "make_rejected_public_skin"
+    And I submit
+  Then I should see "The following skins were updated: public skin"
+  When I follow "Rejected Skins"
+  Then I should see "public skin"
+  When I check "make_unrejected_public_skin"
+    And I submit
+  Then I should see "The following skins were updated: public skin"
+  When I follow "Rejected Skins"
+  Then I should not see "public skin"
+
+  Scenario: An admin can feature a skin
+  Given the approved public skin "public skin"
+    And I am logged in as an admin
+  When I follow "Approved Skins"
+    And I check "make_featured_public_skin"
```

Comment:
Can you tell me what the two skins you're seeing on the Approved Skins page are named? (I'm trying to figure out where they're coming from and why they're not making the other tests fail.)